src/libs/OptionsListUtils.js

```diff
@@ -1282,7 +1282,12 @@ function getOptions(
     if (includePersonalDetails) {
         // Next loop over all personal details removing any that are selectedUsers or recentChats
         _.each(allPersonalDetailsOptions, (personalDetailOption) => {
-            if (_.some(optionsToExclude, (optionToExclude) => optionToExclude.login === personalDetailOption.login)) {
+            if (
+                _.some(
+                    optionsToExclude,
+                    (optionToExclude) => optionToExclude.login === personalDetailOption.login || optionToExclude.login === addSMSDomainIfPhoneNumber(personalDetailOption.login),
```

Comment:
Won't this work as well?

```suggestion
                    (optionToExclude) => optionToExclude.login === addSMSDomainIfPhoneNumber(personalDetailOption.login),
```

Reply:
@rushatgabhane Yeah, your suggestion works as well. Just updated the PR